apps/prairielearn/elements/pl-drawing/pl-drawing.py

```diff
@@ -56,9 +56,9 @@ def check_attributes_rec(element):
     if elements.should_validate_attributes(name):
         try:
             pl.check_attribs(element, required_attribs=[], optional_attribs=attributes)
-        except Exception as e:
-            print(f"Error in {name}: {e}")
-            raise e
+        except Exception as exc:
+            print(f"Error in {name}: {exc}")
```

Comment:
Nit, as long as we're changing this: would this be better written as `raise ... from exc`?

Reply:
Changed to 

```python
        except Exception as exc:
            raise ValueError(f"Error in {name}: {exc}") from exc
```